packages/fiori/src/FlexibleColumnLayout.js

```diff
@@ -82,6 +82,42 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		* Defines the accessibility text for the <code>startColumn</code> region.
+		*
+		* @type {string}
+		* @defaultvalue ""
+		* @public
+		* @since 1.0.0-rc.11
+		*/
+		startColumnAccessibleText: {
```

Comment:
I think accessibleName is the correct term, here it is accessibleText. But, I will have to discuss it with responsible team.